index.html

```diff
@@ -21,110 +21,96 @@ <h3><a href="#" class="navLogo">AYANNAH REUBEN</a></h3>
 		</div>
 			
 		<div class="navResponsive">
-			<h3><a href="#">MENU</a></h3>
+			<h3><a href="#">MENU ⌁</a></h3>
 		</div>
 		
 	</nav>	
 	<!-- /NAV BAR -->
 	
 	
 	
-	
-	
 	<!-- HEADER -- HERO IMAGE AND INTRO SECTION -->		
 	<header>
 		<section class="sectionHeader">
-			<h1 id="headerUpper">Hello</h1>
+			<h1 id="headerUpper">Hello,</h1>
 			<h1 id="headerLower">I'm Ayannah</h1>
 			<h2>I am a full-stack developer</h2>
 		</section>
-		
 	</header>
 	<!-- /HEADER -->
 	
+
 	
-	
-	
-	
-	
-	
-	
-	
-	
-	
-	
-	<!-- MAIN -- ABOUT SECTION -->	
-	<main id="aboutMe">
-		<section class="mainSection">
-			<div>
-				<p>about me</p>
+	<!-- ARTCLE -- ABOUT SECTION -->	
+	<article id="aboutMe">
+		<section class="aboutMeSection">
+			<div class="articleInfo">
+				<h2>ABOUT ME</h2>
+				<p>I am currently enrolled on the Full-Stack Developer course at iO Academy and will be graduating in April 2023.</p>
+				<p>So far, I have learned how to use HTML and CSS for front-end development.</p>
+				<p>Prior to attending the course, I went to University to study Information Technology Management and Business. I also worked in an IT company for 3 years, where my role was a Junior Product Consultant.</p>
+			</div>
+			<div class="articleImage">
```

Comment:
I would use <figure> rather than <div> for image element encapsulation if its just the image.